specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -8851,8 +8858,8 @@
           "description": "Specifies additional base-64 encoded XML formatted information that can be included in the Unattend.xml file, which is used by Windows Setup."
         },
         "patchSettings": {
-          "$ref": "#/definitions/PatchSettings",
-          "description": "Specifies settings related to in-guest patching (KBs)."
+          "$ref": "#/definitions/WindowsPatchSettings",
+          "description": "Specifies settings related to VM Guest Patching (KBs)."
```

Comment:
For consistency with Linux, this description should ideally read: "Specifies settings related to VM Guest Patching on Windows."